swagger.json

```diff
@@ -1,23 +1,29 @@
 {
     "swagger": "2.0",
     "info": {
-        "version": "3.8",
+        "version": "3.9",
         "title": "traccar"
     },
     "host": "traccar.org",
```

Comment:
Can we remove host or maybe change it to `demo.traccar.org`? Because `traccar.org` doesn't actually refer to a Traccar instance.